.eslintrc.json

```diff
@@ -43,5 +44,5 @@
     "pickRandomsFromList": true,
     "respecConfig": true
   },
-  "plugins": ["prettier"]
+  "plugins": ["prettier", "import", "jest"]
```

Comment:
What does import do here?

Reply:
[eslint-plugin-import](uhttps://www.npmjs.com/package/eslint-plugin-import) It's this package.